psl-cli/src/main/java/org/linqs/psl/cli/Launcher.java

```diff
@@ -96,10 +96,10 @@
 
     public static final String MODEL_FILE_EXTENSION = ".psl";
     public static final String DEFAULT_H2_DB_PATH =
-            Paths.get(System.getProperty("java.io.tmpdir"),
-            "cli_" + System.getProperty("user.name") + "@" + getHostname()).toString();
+                    Paths.get(System.getProperty("java.io.tmpdir"),
+                    "cli_" + System.getProperty("user.name") + "@" + getHostname()).toString();
     public static final String DEFAULT_POSTGRES_DB_NAME = "psl_cli";
-    public static final String DEFAULT_IA = MPEInference.class.getName();
+    private static final String DEFAULT_IA = MPEInference.class.getName();
```

Comment:
Why private here?
For constants, there is usually little reason to make it private.